examples/nodejs_wav/Readme.md

```diff
@@ -0,0 +1,48 @@
+# NodeJS voice recognition example using Mozilla DeepSpeech
+
+Install [DeepSpeech](https://github.com/mozilla/DeepSpeech)
+
+```
+pip3 install deepspeech
```

Comment:
So you don't need that step at all, since your `package.json` defines the dependency against deepspeech.

Reply:
Oh I didn't even know.  It sure looks like looks like using pip to install Deepspeech is required to use the NodeJS library because if you come in through NPM (as I did) the Python install steps are the first thing you read:

https://www.npmjs.com/package/deepspeech

Perhaps the first part of the readme should direct NodeJS dev's to skip all the way down to the "NodeJS installation" section.

Reply:
Oh I didn't even know. It sure looks like looks like using pip to install Deepspeech is required because if you come in through NPM (as I did) the Python install steps are the first thing you read:

https://www.npmjs.com/package/deepspeech

Perhaps the first part of the Readme should direct NodeJS dev's to skip all the way down to the "NodeJS installation" section.